pkg/syncer/util.go

```diff
@@ -127,6 +127,7 @@ func IsValidVolume(ctx context.Context, volume v1.Volume, pod *v1.Pod, metadataS
 // if volumeIds is empty, then all volumes from CNS will be retrieved by pagination
 func getQueryResults(ctx context.Context, volumeIds []cnstypes.CnsVolumeId, clusterID string, volumeManager volumes.Manager) ([]*cnstypes.CnsQueryResult, error) {
```

Comment:
Can we refactor and change name of `getQueryResults` to `fullSyncgetQueryResults`?

Reply:
Done. Rename the function.